src/ILLink.RoslynAnalyzer/TrimAnalysis/TrimDataFlowAnalysis.cs

```diff
@@ -80,20 +89,40 @@ public override void TraceVisitBlock (BlockProxy block)
 			if (!trace)
 				return;
 
-			Console.Write ("block " + block.Block.Ordinal + ": ");
+			TraceWrite ("block " + block.Block.Ordinal + ": ", tracingMechanism);
```

Comment:
I think we might as well remove the `tracingMechanism` parameter and just check the field inside `TraceWriteLine`.